☑️  그림자 기능 구현 추가 및 OrderSummaryView에서 프리뷰 설정 코드 삭제

kangminseoung opened the pull request:
1. OrderSummaryViewCell
 - Cell 그림자 기능 구현 설정
 
2. OrderSummaryView
 - 프리뷰 설정 코드 삭제

kangminseoung committed:
✨[FEAT] : OrderSummaryView 그림자 기능 구현 설정 추가

주문 목록에 담기는 Cell에 대한 그림자 기능 구현

kangminseoung committed:
🔥[CHORES] : OrderSummaryView에서 프리뷰를 안보려고 코드 삭제

Quaker-Lee reviewed (approved):
Reviewers 항상 남겨주세요..!
고생하셨습니다

DoyleHWorks committed:
Update OrderSummaryViewCell.swift

layer.shadowOffset = CGSize(width: 0, height: 4)
위 코드를 아래와 같이 변경
layer.shadowOffset = CGSize(width: 0, height: 3)

DoyleHWorks reviewed (approved):
굿입니다~
